package.json

```diff
@@ -96,6 +101,7 @@
     "rehype-raw": "^6.1.1",
     "tailwind-merge": "^2.5.2",
     "tailwindcss-animate": "^1.0.7",
+    "twemoji": "^14.0.2",
```

Comment:
I couldn't find a usage of the package!
Where are we using it?

Reply:
> I couldn't find a usage of the package! Where are we using it?

no use ,I will remove it ,actually chrome browser is not showing flag emoji then I installed it, This is no longer of any use.I will remove it